packages/router/router.ts

```diff
@@ -3126,7 +3163,14 @@ function getMatchesToLoad(
     // If the fetcher path no longer matches, push it in with null matches so
     // we can trigger a 404 in callLoadersAndMaybeResolveData
     if (!fetcherMatches) {
-      revalidatingFetchers.push({ key, ...f, matches: null, match: null });
+      revalidatingFetchers.push({
+        key,
+        routeId: f.routeId,
```

Comment:
Is this just to be more explicit about what ends up on the revalidating fetcher obj? Did we spread too much info before?

Reply:
Correct - no functional change just being more explicit - `f` is a `FetchLoadMatch`:


```ts
interface FetchLoadMatch {
  routeId: string;
  path: string;
}

/**
 * Identified fetcher.load() calls that need to be revalidated
 */
interface RevalidatingFetcher extends FetchLoadMatch {
  key: string;
  match: AgnosticDataRouteMatch | null;
  matches: AgnosticDataRouteMatch[] | null;
}
```